app/components/PredictionMarkets/AddOpinionModal.jsx

```diff
@@ -0,0 +1,173 @@
+                                />
+                            </label>
+                        </Form.Item>
+                        <Form.Item>
+                            <label className="left-label">
+                                <Translate content="prediction.add_opinion_modal.condition" />
+                                <Input
+                                    type="text"
+                                    disabled={true}
+                                    tabIndex={2}
+                                    value={this.props.market.condition}
+                                />
+                            </label>
+                        </Form.Item>
+                        <Form.Item>
+                            <label className="left-label">
+                                <Translate content="prediction.add_opinion_modal.opinion" />
+                            </label>
+                            <Translate
+                                content="prediction.add_opinion_modal.no"
+                                style={{marginRight: "10px"}}
+                            />
+                            <Switch
+                                checked={this.state.bool_opinion}
+                                onChange={this.handleOpinionChange}
+                            />
+                            <Translate
+                                content="prediction.add_opinion_modal.yes"
+                                style={{marginLeft: "10px"}}
+                            />
+                        </Form.Item>
+                        <Form.Item>
+                            <label className="left-label">
+                                <Translate content="prediction.add_opinion_modal.amount" />
+                                <AmountSelector
+                                    onChange={this.handleAmountChange}
+                                    placeholder="0.0"
+                                    tabIndex={6}
+                                    amount={
+                                        this.state.newOpinionParameters.amount
+                                    }
+                                    assets={[
+                                        "1.3.113",
+                                        "1.3.120",
+                                        "1.3.121",
+                                        "1.3.1325",
+                                        "1.3.105",
+                                        "1.3.106",
+                                        "1.3.103"
+                                    ]}
+                                    asset={this.state.selectedAsset}
+                                />
+                            </label>
+                        </Form.Item>
+                    </Form>
+                </div>
+            </Modal>
+        );
+    }
+}
+
+AddOpinionModal.propTypes = {
+    show: PropTypes.bool,
```

Comment:
please stick to commonly used wording `visible`

Reply:
Done